tasks/39_unique-in-order/unique-in-order.js

```diff
@@ -0,0 +1,11 @@
+export function uniqueInOrder(arg) {
+  if (arg === undefined || arg.length === 0) {
+    return [];
+  }
+  arg = [...arg];
```

Comment:
there is no reason to create a copy of array as filter return a new array and don't change original